entrypoint.sh

```diff
@@ -129,8 +146,12 @@ fi
 # Make sure the correct configuration file is used
 server_invocation+=( "-ini" "${CONFIG_FILE}")
 
-# Variable to change the superuser password
-if [[ -n "${MUMBLE_SUPERUSER_PASSWORD}" ]]; then
+if [[ -f /run/secrets/MUMBLE_SUPERUSER_PASSWORD ]]; then
+	#Variable to change the superuser password
+    "${server_invocation[@]}" -supw "$(cat /run/secrets/MUMBLE_SUPERUSER_PASSWORD)"
+    echo "Successfully configured superuser password from container secret"
+elif [[ -n "${MUMBLE_SUPERUSER_PASSWORD}" ]]; then
```

Comment:
How about only setting the `MUMBLE_SUPERUSER_PASSWORD` variable here and keep the modification of the run command in the branch it already happens. Then we don't have to duplicate this functionality

Reply:
Sounds good. Fixed in commit c0ac92af445454e3850827a504baab9f82fa6cd3 as well.